docker-compose.yml

```diff
@@ -8,6 +8,14 @@ services:
     networks:
       - postgres
 
+  grpc-web-proxy:
+    build: ./grpc-web-proxy/
+    ports:
+      - "8080:8080"
+      - "9901:9901"
+    networks:
+      - postgres
```

Comment:
Probably should not be part of the postgres network. I assume we should have a separate "frontend" or "api" network. I think the biggest benefit of the networks is basically only a limited number of services have access to the database (assuming that unlike now in production we do not expose the port).

Reply:
Fixed now. I use "api" network as it represents the purpose best. I also removed the admin port `9901`